ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapper.java

```diff
@@ -67,11 +78,14 @@ public TheirDetails from(Claim.ClaimBuilder builder, CCDDefendant defendant) {
             .defendantId(defendant.getDefendantId());
 
         countyCourtJudgmentMapper.from(defendant.getCountyCourtJudgementRequest(), builder);
+        builder.settlement(settlementMapper.fromCCDDefendant(defendant));
 
         Optional.ofNullable(defendant.getResponseMoreTimeNeededOption()).ifPresent(
             moreTimeNeeded -> builder.moreTimeRequested(moreTimeNeeded.toBoolean())
         );
 
+        builder.settlementReachedAt(defendant.getSettlementReachedAt());
```

Comment:
minor: Would be nice to have this next to line 81: `builder.settlement(settlementMapper.fromCCDDefendant(defendant));`

Reply:
done